docs/source/index.rst

```diff
@@ -0,0 +1,53 @@
+ALFASIM-SDK
+===========
+
+The ALFAsim-SDK help developers to create plugins for an ALFAsim application.
+Plugins can extend the ALFAsim capabilities in a lot of ways, from adding a simple input to a custom calculation of solver equations.
+
+Some examples of the capabilities that can be extended are:
+
+  * Creation of user-defined GUI models, allowing custom models to be available over the Tree and in the Model Explorer.
+  * Registration of additional variables that are not nonlinear system's unknowns, for the ALFAsim solver. Also called "secondary variables"
+  * Customization of the hydrodynamic models, supporting the additions of custom Phases/Fields/Layers.
+  * Support for inclusion of custom mass, momentum and energy equations.
+  * Support for calculating source terms to solver equations.
+  * Support for calculating/updating plugin-defined Simulation Variables
+
+For a list of all capabilities that can be expanded check the :ref:`api-reference-section` in the documentation.
+
+Anyone with programming skills can create a Plugin for ALFAsim, for that, alfasim-SDK provides hooks to customize
+the solver and the user interface. To customize the solver, an API in `C` is provided for application
+written in C/C++, while, for user interface customization, an API in Python allows the developer to customize
+the interface in a declarative way.
```

Comment:
Do you have any suggestions? I don't see how we can include this information without making this paragraph to complex

My idea since the beginning was: "I need to tell everyone that Is easy to create a plugin, you only need python to declare some stuff and C++ for the solver customization"